src/controllers/controller.cpp

```diff
@@ -143,4 +147,6 @@ void Controller::receive(const QByteArray data, mixxx::Duration timestamp) {
         QJSValue incomingDataFunction = m_pEngine->wrapFunctionCode(function, 2);
         m_pEngine->executeFunction(incomingDataFunction, data);
     }
+
```

Comment:
We should deprecate the use of ".incomingData". This will be a nice performance benefit.

Reply:
That is what I did, but [below](https://github.com/mixxxdj/mixxx/pull/2868/files/5a504776092a68a73136dce463dba3e74a78198b#diff-a5ba27f3f0700bcb121ca24fb3b4a837) you are saying the opposite.